### DISCUSSION TITLE
[tvos]Modal background removal

### DISCUSSION THREAD

**Discussion by zsoltbokor3ssprx** [react-native-tvos/react-native-tvos#831] [Feature Requests]
On mobiles the `Modal` component can be configured as transparent, so we can change the backdrop overlay opacity etc.
On tvos the view that's stays in the background of the content has a blurry solid (not transparent) color (something apple default).

Is it possible to omit that background view, so make the Modal's overlay half transparent also on apple tv? 

```
  <Modal
            visible={true}
            transparent
            statusBarTranslucent
            onRequestClose={cancelButtonOnPress}
            supportedOrientations={SUPPORTED_ORIENTATIONS}
        >
          <View style={{backgroundColor: 'rgba(0,0,0,0.5)'}}>....</View>
</Modal>
```

**Comment by istvan-szilagyi** [comment]
+1

**Comment by douglowder** [comment]
Converted this to a feature request discussion, will investigate this soon.

**Comment by RuudBurger** [comment]
@zsoltbokor3ssprx and @istvan-szilagyi this got fixed with #854
